src/workers/test-explorer/logger.ts

```diff
@@ -1,7 +1,115 @@
+import { OutputChannel, window } from "vscode";
+import { TestResult } from "../../model/test-status.enum";
+
+export const OUTPUT_CHANNEL = "Angular Test Explorer";
```

Comment:
Change the name to something like Test Explorer Debug Log, we already have an outputchannel called test explorer to show what is the error when you click on a test, it is weird to have 2 outputchannels called basically the same. this outputchannel It is awesome if you need debug information and in the future we should enable it by user configuration